src/main/java/ch/njol/skript/effects/EffFireworkLaunch.java

```diff
@@ -70,6 +70,8 @@ public boolean init(Expression<?>[] exprs, int matchedPattern, Kleenean isDelaye
 	protected void execute(Event event) {
 		FireworkEffect[] effects = this.effects.getArray(event);
 		int power = lifetime.getOptionalSingle(event).orElse(1).intValue();
+		if (power < 0 || power > 127) // Firework's power range limit
```

Comment:
Remove this condition and let it just do the calculations below so that if someone provides 127+ it will become 127 same for 0 and minus